pkg/spireentry/reconciler.go

```diff
@@ -179,11 +189,34 @@ func (r *entryReconciler) reconcile(ctx context.Context) {
 	}
 }
 
+func (r *entryReconciler) recalculateUnsupportField(ctx context.Context, log logr.Logger) {
+	unsupportedFields, err := r.getUnsupportedFields(ctx)
+	if err != nil {
+		log.Error(err, "failed to get unsupported fields")
+		return
+	}
+
+	if len(unsupportedFields) > 0 {
+		var fields []string
+		for key := range unsupportedFields {
+			fields = append(fields, key)
+		}
+		log.Info("SPIRE Server version does not support fields", "keys", strings.Join(fields, ","))
```

Comment:
This doesn't need to happen now, but it might be nice to track old and new and then only log what changes.